be/src/http/utils.cpp

```diff
@@ -164,14 +164,28 @@ void do_dir_response(const std::string& dir_path, HttpRequest* req) {
     }
 
     const std::string FILE_DELIMETER_IN_DIR_RESPONSE = "\n";
+    const std::string FILE_NAME_SIZE_DELIMETER = "|";
 
     std::stringstream result;
     for (const std::string& file_name : files) {
-        result << file_name << FILE_DELIMETER_IN_DIR_RESPONSE;
+        std::string file_path = dir_path + "/" + file_name;
+
+        if (!type.empty() && type == "V2") {
+            int64_t file_size = -1;
+            auto st = FileSystem::Default()->get_file_size(file_path);
+            if (st.ok()) {
+                file_size = st.value();
+            } else {
+                LOG(WARNING) << "Failed to get file size. file=" << file_name;
+                HttpChannel::send_error(req, HttpStatus::INTERNAL_SERVER_ERROR);
```

Comment:
Should exit function here?

Reply:
According to the original logic, if the file_size cannot be obtained, it will exit this task.

Reply:
so you should `return` after `HttpChannel::send_error(req, HttpStatus::INTERNAL_SERVER_ERROR);`  right?

Reply:
ok, thanks, got it, it has been updated.